IndicoV2.Abstractions/Models/Models/IModelGroup.cs

```diff
@@ -21,7 +21,7 @@ public interface IModelGroup : IModelGroupBase
         /// Gets the ModelGroup status.
         /// </summary>
         /// <value>The status.</value>
-        ModelStatus Status { get; }
+        string Status { get; }
```

Comment:
I think we should deprecate the use of enum ModelStatus and stick to strings (python sdk does this). From my experience, keeping enum makes StrawberryShake type conversion difficult but let me know if I'm missing something here

Reply:
I know it makes it more painful for the conversion, but with the fact that C# is strongly typed, our consumers will generally expect enums. It makes it a lot easier for them to check for errors at compile time, etc.

Reply:
Ah yeah that makes sense

Reply:
Changed back from string to ModelStatus enum